src/docs/product/issues/suspect-commits/index.mdx

```diff
@@ -0,0 +1,70 @@
+redirect_from:
+  - /product/releases/suspect-commits/
+description: "Learn how integrations enable suspect commits."
+---
+
+## Suspect Commits & Suggested Assignees
+
+When you've enabled suspect commits, we can tie together the commits in the release, including:
+
+- Files touched by those commits
+- Files observed in the stack trace
+- Authors of those files
+- The issue number in a commit message
+
+Suspect commits and suggested assignees will then display on the **Issue Details** page in [sentry.io](https://sentry.io):
+
+![The suspect commits and suggested assignees for an issue](suspect-commits-highlighted.png)
+
+## Enable Suspect Commits
+
+This process assumes:
+
+- Source maps — or your platform-specific files for mapping transformed source code to the original source — have been uploaded (check out the docs for your [specific platform](/platforms/))
+
+#### Connect a Repository via integrations
+
+Suspect commits can be enabled via the following integrations:
+- GitHub
+- GitLab
+
+Head to your organization’s **Settings > Integrations** page, and click on the integration you’d like to use. You’ll need to configure the integration first, if you haven’t already and add the repository you'd like to use.
+
+#### Setup Code Mappings
+
+1. Navigate to **Settings > Integrations > GitHub or GitLab > Configurations**.
+
+1. Click the "Configure" button
+
+1. Click the **Code Mappings** tab.
+
+1. Set up a code mapping for each project for which you want to enable. To create a new code mapping, click **Add Mapping**.
+
+1. Fill out the form, then click **Save Changes**. Each form field is described below:
+
+   - **project** (required): This is the Sentry project.
+   - **repo** (required): This is the GitHub repository associated with the Sentry project above. If you have more than one GitHub repository being used per Sentry project, you'll need multiple code mappings.
+   - **branch** (required): This is the default branch of your code we fall back to if you do not have commit tracking set up.
+   - **stack trace root** and **source code root** (optional):
+
+     1. If the file path in your Sentry stack trace frame matches the path to your source code, you do not need to set these values.
+
+        - ex. For example, everything after the branch (`main`) matches the file path of `code.py` using a source code path of `https://github.com/MeredithAnya/testing/blob/main/code.py` so you don't need to set a **stack trace root** and **source code root**.
+
+     1. If the filename in your Sentry stack trace frame doesn't match the path to your source code, you will need to replace the **stack_root** part of the filename with your **source_root** to make the filename match the source code path.
+        - ex. For example, to get `src/code.py` to match `code.py` when the source code path is `https://github.com/MeredithAnya/testing/blob/main/code.py`, change the **stack trace root** to be set as `src/`, and leave **source code root** empty.
```

Comment:
This becomes hard to read as we get into third and fourth level lists. Restructured. Also if these fields are capitalized in the form, they should be capitalized here (I can't see this feature to confirm).

```suggestion

#### Code Mapping Form Fields

- **project** (required) - The Sentry project.
- **repo** (required) - The GitHub repository associated with the Sentry project above. If you have more than one GitHub repository being used per Sentry project, you'll need multiple code mappings.
- **branch** (required) - The default branch of your code we fall back to if you don't have commit tracking set up.
- **stack trace root** and **source code root** (optional):
 - If the file path in your Sentry stack trace frame matches the path to your source code, you don't need to set these values. For example, everything after the branch (`main`) matches the file path of `code.py` using a source code path of `https://github.com/MeredithAnya/testing/blob/main/code.py` so you don't need to set a **stack trace root** and **source code root**.
 - If the filename in your Sentry stack trace frame doesn't match the path to your source code, you'll need to replace the **stack_root** part of the filename with your **source_root** to make the filename match the source code path. For example, to get `src/code.py` to match `code.py` when the source code path is `https://github.com/MeredithAnya/testing/blob/main/code.py`, change the **stack trace root** to be set as `src/`, and leave **source code root** empty.
```